MAINTAINING.md

```diff
@@ -0,0 +1,76 @@
+# RIOT Maintainer Guidelines
+
+This list presents a series of guidelines for maintainers, including technical and non-technical. The list is not exhaustive, and represents a baseline on things that should be ensured for contributions.
+
+Notes:
+- Any of the items in this list might be skipped, if clearly and logically articulated reasons are presented.
+- The order of the steps is meant to maximize efficiency and minimize overhead, with the philosophy that failing in step n makes steps n+x obsolete. However, this efficiency can depend on the quality of the submission itself. If the PR is clearly not in a reviewable state, for example due to poor code cleanliness or overall design, it might be more efficient to give the contributor some broader comments for improvement before further review.
+- This is a working document and any changes, additions, or other discussions are encouraged, via PRs raised against the document.
+
+The list of RIOT maintainers, and the areas of RIOT they each maintain, can be found [here](https://github.com/RIOT-OS/RIOT/wiki/Maintainers).
+
+
+## Technical guidelines
+
+### 1. - Review the fundamentals
+
+1.1 Does the reasoning for this PR make sense? Are the requirements and design goals clearly thought out and clearly expressed? Is the problem that the PR intends to solve clearly stated?  
+1.2 Is the solution presented in the PR as simple as possible to satisfy the requirements, but no simpler?  
+1.3 Is the solution well designed on a high level?  
+1.4 Do the concepts used by the PR make sense?  
+1.5 Does the PR break with existing concepts?  
+1.6 Is the structure of the PR itself valid?  
+1.7 Are there clear and adequate instructions on how to test the PR? This may or may not include implemented tests as part of the PR.  
+1.8 Does the code compile and run?  
+1.9 Does this PR respect the rights of previous authors, either through retaining their commits or by retaining their copyrights in the boilerplate headers?  
+1.10 Is the PR a duplicate of another PR?  
+
+
+### 2. - Review the design of the code
+
+The following list is not exhaustive and addresses the coding issues we have regularly seen in the past. In particular, check that the [Best Practices for RIOT Programming](https://github.com/RIOT-OS/RIOT/wiki/Best-Practice-for-RIOT-Programming) are followed. These checks can be aided (but not replaced) by a tool such as Coccinelle, using the script found in dist/tools/coccinelle.
+
+2.1 Check for code duplication  
```

Comment:
Section above used punctuation. For consistency I would also prefer it to be used here.

Reply:
(same goes for section 4 as well of course)

Reply:
Sorry, just to clarify: do you mean to keep the numbered listing consistent? The paragraphs in sections 2. and 4. indicate a general overview to the section rather than something that would fit in a list format. Perhaps for consistency I could add a short paragraph at the top of section 1.?

Reply:
I've gone with this, also for section 5.